utils/merge.py

```
@@ -1,91 +1,164 @@
+    for line in result.stdout.splitlines():
+        match = re.match(output_pattern, line.strip())
+        if match:
+            dest_path = Path(match.group(1))
+            break
+
+    if dest_path:
+        book.dest_path = str(dest_path)
+        logger.info(f"Output file: {dest_path}")
+    else:
+        # Fallback: construct path from known pattern
+        logger.warning(f"Could not parse output path from Rust output, using fallback")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)